src/commands/env/env-set.js

```diff
@@ -10,42 +14,64 @@ const { log, logJson, translateFromEnvelopeToMongo } = require('../../utils')
  * @returns {Promise<boolean>}
  */
 const envSet = async (key, value, options, command) => {
-  const { api, site } = command.netlify
+  const { context, scope } = options
+
+  const { api, cachedConfig, site } = command.netlify
   const siteId = site.id
 
   if (!siteId) {
     log('No site id found, please run inside a site folder or `netlify link`')
     return false
   }
 
-  const siteData = await api.getSite({ siteId })
+  const { siteInfo } = cachedConfig
```

Comment:
Using `cachedConfig.siteInfo` instead of refetching from `api.getSite()` -- this is safe to do, yes?